physionet-django/notification/templates/notification/email/mailto_contact_applicant.html

```diff
@@ -0,0 +1,43 @@
+{% load i18n %}{% autoescape off %}
```

Comment:
The License/DUA shouldn't really be hardcoded into the email. Makes it tricky to keep the email in sync with the system documents.